test/Microsoft.AspNetCore.Antiforgery.Test/DefaultAntiforgeryTest.cs

```diff
@@ -40,7 +40,7 @@ public class DefaultAntiforgeryTest
         }
 
         [Fact]
-        public void ChecksSSL_ValidateTokens_Throws()
+        public void ChecksSSL_TryValidateTokenSet_Throws()
```

Comment:
Name was better before. `TryValidateTokenSet` get's called but that method certainly isn't throwing.